src/jackdaw/streams/interop.clj

```diff
@@ -323,7 +333,20 @@
     (clj-kstream
      (.transform ^KStream kstream
                  ^TransformerSupplier (transformer-supplier transformer-supplier-fn)
-                 ^"[Ljava.lang.String;" (into-array String state-store-names))))
+                 ^"[Ljava.lang.String;" (into-array String
```

Comment:
is that how a string array is annotated? 🤯

Reply:
Yeah, although arrays are objects in Java they are not an instance of a class, so have a slightly funky type signature :-)